mods/d2k/weapons/missiles.yaml

```diff
@@ -32,147 +33,105 @@ Bazooka:
 		Explosions: tiny_explosion
 		ImpactSounds: EXPLSML1.WAV
 
+^Missile:
+	Inherits: ^Rocket
+	ReloadDelay: 60
+	Range: 5c512
+	MinRange: 1c0
+	Projectile: Missile
+		Blockable: false
+		Shadow: true
+		HorizontalRateOfTurn: 3
+		RangeLimit: 6c614
+		Inaccuracy: 384
+		CruiseAltitude: 1c0
+		MinimumLaunchAngle: 64
+		VerticalRateOfTurn: 10
+		Image: MISSILE2
+		TrailImage: large_trail
+		Speed: 320
+	Warhead@1Dam: SpreadDamage
+		Spread: 256
+		Damage: 480
+		Versus:
+			none: 15
+			wall: 75
+			building: 60
+			wood: 65
+			light: 90
+			heavy: 100
+			invulnerable: 0
+			cy: 30
+			harvester: 50
+	Warhead@3Eff: CreateEffect
+		Explosions: small_explosion
+
+Bazooka:
+	Inherits: ^Rocket
```

Comment:
Ah, I see.  Would it make sense to change these to an arcing Bullet like we use for non-guided projectiles in RA and TD?